tests/jest/config/platform-properties.json

```diff
@@ -119,30 +119,34 @@
-  "modal.ansible.scheduleMode.manual.info": "Manual make this automation immediately run once. Once it runs, it will be set to disabled.",
-  "modal.ansible.scheduleMode.once.title": "Run once mode",
-  "modal.ansible.scheduleMode.once.info": "When this policy is violated, the automation will run one time. Once it runs, it will be set to disabled.",
-  "modal.ansible.scheduleMode.disabled.title": "Disable automation",
-  "modal.ansible.scheduleMode.disabled.info": "Automation won't run until you change this setting.",
+  "ansible.automation.heading.create": "Create policy violation automation",
+  "ansible.automation.heading.edit": "Edit policy violation automation",
+  "ansible.automation.description.create": "Create new provider credentials to automate the selected policy and violation. You can also schedule the automation to run once, continuously, or disable.",
+  "ansible.automation.description.edit": "Modify existing provider credentials to automate the selected policy and violation. You can also schedule the automation to run once, continuously, or disable.",
+  "ansible.launch.connection": "Launch to Ansible URL",
+  "ansible.credential.selection.title": "Credential",
+  "ansible.credential.selection.placeholder": "Select Ansible credential",
+  "ansible.no.credential": "Ansible credential is required to create an Ansible job. Create your console by clicking the following link:",
+  "ansible.launch.createCredential": "Create new provider credentials",
+  "ansible.jobTemplates.selection.title": "Job template",
+  "ansible.jobTemplates.selection.placeholder": "Select Ansible job template",
+  "ansible.jobTemplates.editor.title": "Extra variables in the Ansible job template",
+  "ansible.jobTemplates.editor.placeholder": "Enter extra variables from the Ansible job template here",
+  "ansible.no.jobTemplates.info": "Failed to get Ansible job template information. Check the related Ansible Tower and Ansible job template settings.",
+  "ansible.scheduleMode.title": "Schedule automation",
+  "ansible.scheduleMode.info": "Choose how often you want this automation to run. You can also choose to manually run it, which saves your change as is.",
+  "ansible.scheduleMode.manual.title": "Manual run",
+  "ansible.scheduleMode.manual.info": "Manually set this automation to run once. After the automation runs, it is set to disabled.",
+  "ansible.scheduleMode.once.title": "Run once mode",
+  "ansible.scheduleMode.once.info": "When this policy is violated, the automation runs one time. After the automation runs, it is set to disabled.",
+  "ansible.scheduleMode.disabled.title": "Disable automation",
+  "ansible.scheduleMode.disabled.info": "Automation does not run until this setting is updated.",
+  "ansible.unsaved.data": "You made changes to the configuration. Do you want to save your policy automation? Click the Cancel button to close.",
+  "ansible.loading.info": "Loading policy violation automation data...",

```

Comment:
@dockerymick This is the loading message with spinner when first time open the panel and still loading backend data